config/env/default.js

```diff
@@ -28,5 +28,8 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: 'modules/core/client/img/brand/favicon.ico',
+  uploads: {
+    profileUploadDir: './modules/users/client/img/profile/uploads/'
```

Comment:
@ilanbiala 
Seems to me that favicon.ico is using ```./``` and isn't using ```config.favicon```
https://github.com/meanjs/mean/blob/master/config/lib/express.js#L68
Maybe, we need to adjust this.

@lirantal 
I can't understand your expectations, can you give more details?